### PR TITLE
chore: add isHostedOnCloud property in NPS request

### PR DESCRIPTION
**Pull request by maccomaccomaccomacco**
### What does it do?

The NPS score needs to include information about whether the project is run on Cloud or not.

### Why is it needed?

We want to track feedback specifically coming from the projects run on Cloud.

### How to test it?

Actually, I'm struggling to test this on Cloud.
I tested it only on local:
- run the getstarted project
- make the NPS panel appear
- send the feedback
- check the request sent, it includes the `onCloud` property

### CODE REVIEW

File: packages/core/admin/admin/src/components/NpsSurvey.tsx [hunk 1]
```diff
@@ -189,6 +189,7 @@ const NpsSurvey = () => {
         environment: currentEnvironment,
         version: strapiVersion ?? undefined,
         license: window.strapi.projectType,
+        onCloud: process.env.STRAPI_HOSTING === 'strapi.cloud'
```
Comment (innerdvations) [inline packages/core/admin/admin/src/components/NpsSurvey.tsx:192]: Should we use `isHostedOnStrapiCloud` to match the name [here](https://github.com/strapi/strapi/blob/develop/packages/cli/create-strapi-app/src/utils/usage.ts#L40)?

Comment (jhoward1994) [reply packages/core/admin/admin/src/components/NpsSurvey.tsx]: very nit picky but I prefer isXYZ as a naming convention for booleans like this, happy to go with this though

Comment (maccomaccomaccomacco) [reply packages/core/admin/admin/src/components/NpsSurvey.tsx]: well, if it's two of you calling for it, I should change it

Comment (maccomaccomaccomacco) [reply packages/core/admin/admin/src/components/NpsSurvey.tsx]: done

**Comment by dimitrinicolas** [comment]
Hey @maccomaccomaccomacco, I appreciate the mention, but I'm not actually part of the Strapi team. I think you might have tagged me by mistake. Have a great day!

**Comment by maccomaccomaccomacco** [comment]
Sorry @dimitrinicolas 😅

@Jimimimi 👉

**Comment by Jimimimi** [comment]
Hey @jhoward1994 @maccomaccomaccomacco, correct you'd need to publish an experimental version, and then you can use the cloud CLI to deploy it to a cloud project 

- `npx create-strapi-app@0....`
- Signup / Login to cloud when prompted
- `npm run deploy` from within your strapi base dir

**Comment by jhoward1994** [comment]
Thanks @Jimimimi, I can try and test that out on the cloud

Are the formatting changes here intentional? Looks like it was done by this commit @maccomaccomaccomacco 

[`a68d1b0` (#21849)](https://github.com/strapi/strapi/pull/21849/commits/a68d1b0a9167a3f7ad17ea08807c038c57fceafb)

**Comment by maccomaccomaccomacco** [comment]
just run `yarn prettier --write .` @jhoward1994 🤷🏼‍♂️ as Ben asked

**Review by jhoward1994 — APPROVED** [review]

**Review by innerdvations — APPROVED** [review]
LGTM

**Pull request merged**